From: =?utf-8?q?Guido_G=C3=BCnther?= <agx@sigxcpu.org>
Date: Sat, 12 Apr 2025 14:24:30 +0200
Subject: app-grid: Fix missing icon
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

emblem-ok-symbolic got remove in the 48 adwaita icon theme:

See https://gitlab.gnome.org/GNOME/adwaita-icon-theme/-/commit/a00466f12bd5419d6f22c1097a16fd670431c9d4

Signed-off-by: Guido Günther <agx@sigxcpu.org>
Part-of: <https://gitlab.gnome.org/World/Phosh/phosh/-/merge_requests/1701>
---
 src/app-grid.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/app-grid.c b/src/app-grid.c
index 96b5298..91cdc13 100644
--- a/src/app-grid.c
+++ b/src/app-grid.c
@@ -560,8 +560,8 @@ on_folder_edit_toggled (PhoshAppGrid *self, GtkToggleButton *toggle_btn)
     const char *folder_name = phosh_folder_info_get_name (priv->open_folder);
     gtk_entry_set_text (GTK_ENTRY (priv->folder_name_entry), folder_name);
     gtk_widget_grab_focus (priv->folder_name_entry);
-    gtk_image_set_from_icon_name (GTK_IMAGE (priv->folder_name_img),
-                                  "emblem-ok-symbolic", GTK_ICON_SIZE_BUTTON);
+    gtk_image_set_from_icon_name (GTK_IMAGE (priv->folder_name_img), 
+                                  "checkmark-symbolic", GTK_ICON_SIZE_BUTTON);
   } else {
     const char *folder_name = gtk_entry_get_text (GTK_ENTRY (priv->folder_name_entry));
     if (gm_str_is_null_or_empty (folder_name))
